app/presenters/blacklight/facet_item_presenter.rb

```diff
@@ -46,7 +46,7 @@ def label
         facet_config.query[value][:label]
       elsif facet_config.date
         localization_options = facet_config.date == true ? {} : facet_config.date
-        I18n.l(Time.zone.parse(value), localization_options)
+        I18n.l(Time.zone.parse(value), **localization_options)
```

Comment:
Do we need to require i18n >= 1.7.0 in our gemspec (when named params were added to the i18n gem), or otherwise handle both?

Reply:
Good idea.